classes/shortcodes.php

```diff
@@ -248,13 +292,20 @@ function get_directions_link( $post ) {
 
 			if ( $pm = get_metadata( 'post', $post ) ) {
 
-				$address .= empty( $pm['location_address'][0] ) ? '' : $pm['location_address'][0];
-				$address .= empty( $pm['location_city'][0] ) ? '' : ' ' . $pm['location_city'][0];
-				$address .= empty( $pm['location_state'][0] ) ? '' : ' ' . $pm['location_state'][0];
-				$address .= empty( $pm['location_zip'][0] ) ? '' : ' ' . $pm['location_zip'][0];
-				$address .= empty( $pm['location_country'][0] ) ? '' : ' ' . $pm['location_country'][0];
-
-				$directions_url = 'http://google' . $options['default_domain'] . '/maps?saddr=&daddr=' . urlencode( $address );
+				$address .= empty( $pm['location_address'][0] ) ? ''
+					: $pm['location_address'][0];
+				$address .= empty( $pm['location_city'][0] ) ? ''
+					: ' ' . $pm['location_city'][0];
+				$address .= empty( $pm['location_state'][0] ) ? ''
+					: ' ' . $pm['location_state'][0];
+				$address .= empty( $pm['location_zip'][0] ) ? ''
+					: ' ' . $pm['location_zip'][0];
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->